packages/delivery-service/src/persistence/metrics/setMetrics.ts

```diff
@@ -7,6 +7,11 @@ export function countMessage(redis: Redis) {
         messageSizeBytes: number,
         deliveryServiceProperties: DeliveryServiceProperties,
     ) => {
+        if (deliveryServiceProperties.metricsRetentionDurationInSeconds === 0) {
```

Comment:
What happens if the number is negative? Does the schema validation prevent this? Otherwise, it might make sense to use deliveryServiceProperties.metricsRetentionDurationInSeconds <= 0 instead of deliveryServiceProperties.metricsRetentionDurationInSeconds === 0

Reply:
good hint, changed to <=